plugins/index.js

```diff
@@ -0,0 +1,29 @@
+const github = require('./github.js');
+const httpcat = require('./httpcat.js');
+const jenkins = require('./jenkins.js');
```

Comment:
This is better, but I was actually thinking of using https://nodejs.org/docs/latest-v5.x/api/fs.html#fs_fs_readdirsync_path to find plugins at start-time.